internal/attestation/measurements/measurements_test.go

```diff
@@ -321,3 +321,95 @@ func TestMeasurementsFetchAndVerify(t *testing.T) {
 		})
 	}
 }
+
+func TestAllBytes(t *testing.T) {
+	testCases := map[string]struct {
+		b       byte
+		wantPCR []byte
+	}{
+		"0x00": {
+			b:       0x00,
+			wantPCR: []byte{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
```

Comment:
Why not use` bytes.Repeat` here?

Reply:
Since this code tests `AllBytes` (now `PCRWithAllBytes`) which does the repetition I wanted to show the full expected value once.

Reply:
Imo you can assume these (especially external/stdlib) dependencies to be correct, as they don't belong to your _system under test_.